nav2_behavior_tree/src/generate_nav2_tree_nodes_xml.cpp

```diff
@@ -24,12 +23,31 @@
 
 #include "plugins_list.hpp"
 
+static void split_string(
+  std::vector<std::string> & result,
+  const std::string & input,
+  const std::string & isAnyOf)
+{
+  std::string::size_type start = 0;
+  std::string::size_type end = input.find_first_of(isAnyOf, start);
+
+  while (end != std::string::npos) {
+    result.push_back(input.substr(start, end - start));
+    start = end + 1;
+    end = input.find_first_of(isAnyOf, start);
+  }
+
+  if (start < input.length()) {
+    result.push_back(input.substr(start));
+  }
+}
+
 int main()
 {
   BT::BehaviorTreeFactory factory;
 
   std::vector<std::string> plugins_list;
-  boost::split(plugins_list, nav2::details::BT_BUILTIN_PLUGINS, boost::is_any_of(";"));
+  split_string(plugins_list, nav2::details::BT_BUILTIN_PLUGINS, ";");
```

Comment:
See https://github.com/ros-navigation/navigation2/pull/4492: I replace with an existing nav2_util as part of fixing up some Jazzy release issues w.r.t. boost removal

Reply:
Got it, will go looking for this function.

Reply:
This should be fixed in the latest push now.